source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -587,6 +588,10 @@ def __init__(self):
         parser_get.add_argument("traveler_type", type=option_value(acceptable_traveler_types))
         parser_get.add_argument("_override_scenario", type=str, description="debug param to specify a custom scenario")
 
+        parser_get.add_argument("walking_transfer_penalty", type=int)
```

Comment:
I don't think so, for me they are only instances params (or maybe hidden ones for test purpose ?)